LICENSE

```diff
@@ -0,0 +1,27 @@
+Business Source License 1.1
+License text copyright (c) 2017 MariaDB Corporation Ab, All Rights Reserved. "Business Source License" is a trademark of MariaDB Corporation Ab.
+-----------------------------------------------------------------------------
+Parameters
+Licensor: Levana Foundation
+Licensed Work: [Levana Predict]; the Licensed Work is (c) 2024 Levana Foundation
+Additional Use Grant: Any uses listed and defined at: https://github.com/Levana-Protocol/levana-predict/blob/main/LICENSE
+Change Date: The earlier of July 16, 2025 or a date specified at: https://github.com/Levana-Protocol/levana-predict/blob/main/LICENSE
```

Comment:
is the date here ok?

Reply:
Good catch. I'll set this to a year from today instead.

Reply:
Sorry, two years, that was the original criteria.